server/cli/launch.py

```diff
@@ -164,14 +168,27 @@ def launch(
     else:
         port = find_available_port(host)
 
+    if about:
+        def url_check(url):
+            try:
+                result = urlparse(url)
+                if all([result.scheme, result.netloc]):
+                    return True
+                else:
+                    return False
+            except ValueError:
+                return False
+        if not url_check(about):
+            raise click.ClickException("Incorrect URL specified for --about. (Example format: http://example.com)")
```

Comment:
Error message could be improved.   What is incorrect?   Suggest something like:   `--about must specify an absolute URL`

Also, consider if you want to mandate that scheme is `http`, rather than letting it be any URI scheme (such as file:, ftp:, etc).   If you want the UI to mandate HTTP, the help and error message need to reflect this.